src/components/input/Input.tsx

```diff
@@ -33,6 +36,7 @@ export const Input = React.forwardRef<HTMLInputElement, Props>(
           type={type}
           popup={popup}
           small={small}
+          dark={savedThemeValue === THEME_OPTIONS.DARK}
```

Comment:
I think usage of ThemeProvider from styled-components might be handy here, we would avoid prop-drilling

https://dev.to/aromanarguello/how-to-use-themes-in-styled-components-49h